build.gradle

```diff
@@ -243,59 +243,58 @@ task codeCoverageReport(type: JacocoReport) {
     }
 }
 
+// https://docs.gradle.org/6.7.1/userguide/publishing_maven.html
```

Comment:
What does this comment help for?

Reply:
ah it's just a note for writing this PR. I'll remove it.